db.json

```diff
@@ -511,6 +511,44 @@
         "type": "pcdsdevices.happi.containers.LCLSItem",
         "z": 742.4427
     },
+    "comp_motor": {
```

Comment:
If I'm allowed to be pedantic in these reviews, this is a rough name for a motor- high chance for cross-hutch name collisions. I'm not worried though since we can rename things later with low impact.

Reply:
We should come up with a way to follow up on these things and work with the maintainer (?) of the device.
For now, maybe we make issues here and ping that person / open Jira tickets?

Reply:
-> https://github.com/pcdshub/device_config/issues/43